staff_features/accessions/accession_delete.feature

```diff
@@ -0,0 +1,23 @@
+Feature: Accession Delete
+  Background:
+    Given an administrator user is logged in
+      And an Accession has been created
+  Scenario: Accession is deleted from the search results
+     When the user clicks on 'Browse'
+      And the user clicks on 'Accessions'
+      And the user filters by text with the Accession title
+      And the user checks the checkbox of the Accession
+      And the user clicks on 'Delete'
+      And the user clicks on 'Delete Records'
+     Then the 'Records deleted' message is displayed
```

Comment:
See prior comment on Accession Create scenario

Reply:
Same as prior comment, I have added a "Then" clause to verify that the accession is actually deleted.